doc/dependency_decisions.yml

```diff
@@ -58,29 +58,37 @@
     :versions: []
     :when: 2023-05-01 18:16:13.435000538 Z
 - - :license
-  - "@viamrobotics/eslint-config"
+  - '@viamrobotics/eslint-config'
   - Apache 2.0
   - :who:
     :why: 'TODO(RSDK-583): why is license finder unable to detect this license automatically?'
     :license_links: https://github.com/viamrobotics/js-config/blob/main/LICENSE
     :versions: []
     :when: 2023-05-01 18:17:50.394914571 Z
 - - :license
-  - "@viamrobotics/rpc"
+  - '@viamrobotics/rpc'
   - Apache 2.0
   - :who:
     :why: 'TODO(RSDK-583): why is license finder unable to detect this license automatically?'
     :license_links: https://github.com/viamrobotics/goutils/blob/main/LICENSE
     :versions: []
     :when: 2023-05-01 18:19:05.224492180 Z
 - - :license
-  - "@viamrobotics/typescript-config"
+  - '@viamrobotics/typescript-config'
   - Apache 2.0
   - :who:
     :why: 'TODO(RSDK-583): why is license finder unable to detect this license automatically?'
```

Comment:
I don't know what this file is, but I know the answer to this question and how to fix it, if you want to assign this ticket to me!

Reply:
This file let's us manually approve libraries. I'll assign the ticket to you for now! I think it's more broad than just JS stuff, but if you could tackle the JS bit (and maybe explain why?), we might be able to fix the other SDK issues ourselves

Reply:
Nice! For the JS side, it's a rather trivial packaging issue that switching to pnpm workspaces resolves